flow-nodes/api-builder-plugin-format-date/package.json

```diff
@@ -9,7 +9,7 @@
 		"api-builder"
 	],
 	"engines": {
-		"node": ">= 10.0"
+		"node": ">= 12.13"
```

Comment:
```suggestion
		"node": ">= 12.15"
```

Reply:
The plugin template comes with engine >= 12.15, hence the suggestion for alignment with what we have there